lib/web_ui/test/text_editing_test.dart

```diff
@@ -16,36 +17,46 @@ const MethodCodec codec = JSONMethodCodec();
 
 TextEditingElement editingElement;
 EditingState lastEditingState;
+String lastInputAction;
 
-final InputConfiguration singlelineConfig =
-    InputConfiguration(inputType: EngineInputType.text);
+final InputConfiguration singlelineConfig = InputConfiguration(
+  inputType: EngineInputType.text,
+  obscureText: false,
+  inputAction: 'TextInputAction.done',
+);
 final Map<String, dynamic> flutterSinglelineConfig =
     createFlutterConfig('text');
 
-final InputConfiguration multilineConfig =
-    InputConfiguration(inputType: EngineInputType.multiline);
+final InputConfiguration multilineConfig = InputConfiguration(
+  inputType: EngineInputType.multiline,
+  obscureText: false,
+  inputAction: 'TextInputAction.newline',
+);
 final Map<String, dynamic> flutterMultilineConfig =
     createFlutterConfig('multiline');
 
 void trackEditingState(EditingState editingState) {
   lastEditingState = editingState;
 }
 
+void trackInputAction(String inputAction) {
+  lastInputAction = inputAction;
+}
+
 void main() {
+  tearDown(() {
+    lastEditingState = null;
+    lastInputAction = null;
+  });
+
   group('$TextEditingElement', () {
     setUp(() {
       editingElement = TextEditingElement(HybridTextEditing());
     });
 
     tearDown(() {
-      try {
+      if (editingElement.isEnabled) {
```

Comment:
question: what happens if we try to disable a not enabled text field? (Can we add a comment to the tear down so we can remember later :) )